_data/committers.yml

```diff
@@ -332,6 +332,14 @@
   role: Committer
   alias: mingmwang
   affiliation: Ebay
+- name: Mehmet Ozan Kabak
```

Comment:
@ozankabak and @mustafasrepo  -- can you verify that this is the information you would like listed on the site?

Thank you

Reply:
These informations are correct. Thanks @alamb .